### PR TITLE
Reorganizing the Unity documents and adding Unity Organization document

### PR DESCRIPTION
**Pull request by KayleeLChan**
I noticed there were a few Unity documents (Unity_Intro, Unity_ML_Agents, Unity_Tutorial_2D_TopDown_Character_Controller, and Unity_UI), so I decided to compile them into a Unity subdirectory under Tech_Stacks. I also created a guide on how to organize Unity projects.

To the reviewer, please read through the draft in Topics/Tech_Stacks/Unity/Unity_Organization.md. Images used inside the document can be found inside the Unity_Organization_Images folder. Keep an eye out for possible typos! ;) Please let me know if there are any areas where clarification could be useful or if there are any points that are not clear/easy to follow. Anyone who is brand new to Unity and who read the Intro to Unity document should be able to follow along with this guide!

### CODE REVIEW

File: Topics/Tech_Stacks/Unity/Unity_Organization.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+|--|--|
+|![](Unity_Organization_Images/before-hierarchy.png)|![](Unity_Organization_Images/after-hierarchy.png)
+
+## Organizing your prefabs
+If you are developing a game in Unity, then you will inevitably need to use prefabs. Especially as your game grows and becomes more complex, you will find yourself needing to use prefabs and prefab variants to efficiently make changes and create consistent gameplay.
+
+### Brief introduction to prefabs
+Prefab assets act as templates for objects in your game. Rather than manually having to edit identical objects many times over if they appear frequently enough in your game, it is easier to have a single prefab asset in your project that objects in your game are instances of. Any changes made to your prefab will be reflected in all of its instances. For example, if you have a sign in your game, but realise you gave it the wrong material, rather than having to apply the new material to each and every sign in your game, you can apply the new material to your sign prefab and each sign instance in your game will automatically update to that new material.
+
+![](Unity_Organization_Images/prefab-demo.gif)
```
Comment (paridhi26) [inline Topics/Tech_Stacks/Unity/Unity_Organization.md:135]: The gifs are really helpful!

File: Topics/Tech_Stacks/Unity/Unity_Organization.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
+# Introduction to Unity Organization
+
+## Table of contents
```
Comment (paridhi26) [inline Topics/Tech_Stacks/Unity/Unity_Organization.md:3]: Well organized and easy to navigate :)

**Review by paridhi26 — APPROVED** [review]
I am new to Unity and found this document easy to follow. This is good for merging!

File: Topics/Tech_Stacks/Unity/Unity_Organization.md (hunk 2 above)
Comment (KayleeLChan) [reply Topics/Tech_Stacks/Unity/Unity_Organization.md:3]: Awesome, thanks Paridhi! 💃 💃 💃

**Comment by paridhi26** [comment]
Moving files to a different folder is breaking some path links in Teck_Stacks.md

**Pull request merged**